src/applications/gi/components/vet-tec/VetTecApprovedProgramsList.jsx

```diff
@@ -1,102 +1,145 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import ContactInformation from '../profile/ContactInformation';
 import { formatCurrency, isPresent } from '../../utils/helpers';
+import classNames from 'classnames';
 
-function VetTecApprovedProgramsList({
-  selectedProgram,
-  institution: { programs },
-}) {
-  if (programs && programs.length) {
-    const programRows = programs.map((program, index) => {
-      const programLength =
-        isPresent(program.lengthInHours) && program.lengthInHours !== '0'
-          ? `${program.lengthInHours} hours`
-          : 'TBD';
-      const tuition = isPresent(program.tuitionAmount)
-        ? formatCurrency(program.tuitionAmount)
-        : 'TBD';
-      const checked =
-        selectedProgram &&
-        program.description.toLowerCase() === selectedProgram.toLowerCase();
+class VetTecApprovedProgramsList extends React.Component {
```

Comment:
i'm going and ahead switching this over to component with a state in prep for https://github.com/department-of-veterans-affairs/va.gov-team/issues/9722